component-classes/shortcuts.js

```diff
@@ -1,15 +1,17 @@
+import { button } from './index.js';
```

Comment:
I'm not sure but wouldn't this make the `buttons` shortcuts bundle increase in size, because it would need to include the whole index.js, too? @imprashast you might have a better understanding of it.

Reply:
I am not really sure how these are used? 🤔  But I figure they are just used as strings... and it won't affect any bundle size? 🤷‍♂️

Reply:
Yeah, you're right. The bundle size will increase, but won't these shortcuts be used build time and not run time?

Reply:
Yes, they should... 🤷

Reply:
Used here:
https://github.schibsted.io/finn/frontpage-podium/blob/3c870cda1daa9cf8c83b0d27a7fb0c3d4df8f5ad/uno.config.js
...and here:
https://github.schibsted.io/finn/consent-frontend/blob/4b1325e32d88cf2d082026580856e65d97c6fe3c/layouts/frontpage/uno.config.js